### PR TITLE
Use CSS Vars for Colors

### PR DESCRIPTION
**Pull request by chrisvire**
Scripture App Builder has been changed to generate sab-app.css with CSS vars for colors. Also, clean up where items are added to the `<head>`

The example_data folder needed to be updated for new CSS and I am not sure what the previous project was used. So I updated the project and included a README with the Google Drive location.

### CODE REVIEW

File: static/override-sab.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
```
Comment (chrisvire) [inline static/override-sab.css:1]: SAB stopped excluding margin-top, margin-bottom which was a bit of a hack.  We should exclude it from inside the PWA.

**Comment by chrisvire** [comment]
@FyreByrd Could you look at the main README.md?

**Review by FyreByrd — CHANGES_REQUESTED** [review]
There is one line in `README.md` that needs to be updated to reflect the changes in `package.json`, other than that, this looks fine.

**Review by FyreByrd — APPROVED** [review]
Looks good to me

**Review by davidmoore1 — APPROVED** [review]

**Pull request merged**